### PR TITLE
Add remaining cards to Station One data pack

### PR DESCRIPTION
**Pull request by lostie**
Spoiler link: https://imgur.com/a/C3aOF#UjtQ1rh

### CODE REVIEW

File: pack/so.json [hunk 1]
```diff
@@ -73,6 +159,60 @@
...
+        "faction_cost": 3,
+        "flavor": "\"Submit.\"",
+        "illustrator": "Ethan Patrick Harris",
+        "keywords": "Code Gate - Bioroid - AP",
+        "pack_code": "so",
+        "position": 30,
+        "quantity": 3,
+        "side_code": "corp",
+        "strength": 4,
+        "text": "The runner can spend [click][click] to break up to 2 subroutines on Nerine 2.0.\n[subroutine] Deal 1 brain damage. The Corp may draw 1 card.\n[subroutine] Deal 1 brain damage. The Corp may draw 1 card.",
```
Comment (JoelCFC25) [inline pack/so.json]: capitalize to Runner

File: pack/so.json [hunk 2]
```diff
@@ -90,6 +230,60 @@
...
+        "faction_code": "nbn",
+        "faction_cost": 3,
+        "flavor": "\"The customer pays to use our service, and then the advertisers pay us to put ads on their screens, and then the customers pay us a premium to remove the ads. Welcome to the dream.\"",
+        "illustrator": "Andreas Zafiratos",
+        "keywords": "Advertisement",
+        "pack_code": "so",
+        "position": 34,
+        "quantity": 3,
+        "side_code": "corp",
+        "text": "THe first time the Runner spends [click] to gain 1[credit] each turn (not through a card effect), gain 1[credit].",
```
Comment (JoelCFC25) [inline pack/so.json]: THe -> The

**Comment by Alsciende** [comment]
Thank you Sergio.

File: pack/so.json (hunk 1 above)
Comment (Alsciende) [inline pack/so.json]: thank you Joel

File: pack/so.json (hunk 2 above)
Comment (Alsciende) [inline pack/so.json]: thank you Joel

File: pack/so.json (hunk 1 above)
Comment (lostie) [inline pack/so.json]: Here I left it as per the card's wording. It's the recurring inconsistency in the card templates FFG is using :/

**Comment by lostie** [comment]
Thanks for reviewing @Alsciende @JoelCFC25 🥇

File: pack/so.json (hunk 1 above)
Comment (Alsciende) [inline pack/so.json]: We're not above fixing card texts here and there :)

**Pull request merged**